### ISSUE TITLE
Zotero API v3 on the horizon

### ISSUE THREAD

**Issue by jlegewie** [LaTeXing/LaTeXing#27]
Hi,

I just wanted to point the developer to future revisions and improvements to the zotero server API. The revised API will support all-JSON responses, which might be easier to handle from Python. Here is relevant post on the Zotero dev list:

https://groups.google.com/forum/#!topic/zotero-dev/Ay7_a9qsp_U

**Comment by sunew** [comment]
Hi, any news on this one?

**Comment by csch0** [comment]
Oh pushed this to the back of my todo list since its still working and not deprecated at the moment. Anyway its still on the list.. so stay tuned. :+1:

**Comment by sunew** [comment]
thanks:) Related: I would like to use biblatex instead of bibtex format, but it seems the Remote.bib entries from Zotero are using the bibtex format. Anyway to work around this?

**Comment by csch0** [comment]
Most of the times using biblatex for myself. So whats the problem here?

**Comment by sunew** [comment]
When exporting manually from Zotero you can choose between the biblatex and the bibtex translator. 
It seems the Remote.bib entries fetched via the api are in the bibtex format, not the biblatex format?

**Comment by sunew** [comment]
I'm thinking about manually exporting the zotero library, as biblatex, to a global bib file, and pointing pdflatex to that file. But it would be very nice just to be able to get the remote.bib in biblatex format with the zotero integration of latexing

**Comment by csch0** [comment]
You could use the global bibliography option of LaTeXing too.

Could you drop an email at info[at]latexing.com would have some questions relating that and its a bit off-topic.

**Comment by sunew** [comment]
yep :) I'll play around a little and let you know what I come up with.

**Comment by sunew** [comment]
I mailed you with more info.
